jetty-documentation/src/main/asciidoc/administration/sessions/session-configuration-hazelcast.adoc

```diff
@@ -0,0 +1,184 @@
+  + https://hazelcast.org/
+  + http://www.apache.org/licenses/LICENSE-2.0.html
+
+Proceed (y/N)? y
+INFO  : server          transitively enabled, ini template available with --add-to-start=server
+INFO  : sessions        transitively enabled, ini template available with --add-to-start=sessions
+INFO  : session-store-hazelcast-remote initialized in ${jetty.base}/start.d/session-store-hazelcast-remote.ini
+MKDIR : /Users/olamy/mvn-repo/com/hazelcast/hazelcast/3.8.2
+DOWNLD: http://central.maven.org/maven2/com/hazelcast/hazelcast/3.8.2/hazelcast-3.8.2.jar to /Users/olamy/mvn-repo/com/hazelcast/hazelcast/3.8.2/hazelcast-3.8.2.jar
+MKDIR : ${jetty.base}/lib/hazelcast
+COPY  : /Users/olamy/mvn-repo/com/hazelcast/hazelcast/3.8.2/hazelcast-3.8.2.jar to ${jetty.base}/lib/hazelcast/hazelcast-3.8.2.jar
+COPY  : /Users/olamy/mvn-repo/com/hazelcast/hazelcast-client/3.8.2/hazelcast-client-3.8.2.jar to ${jetty.base}/lib/hazelcast/hazelcast-client-3.8.2.jar
+INFO  : Base directory was modified
+----
+
+Doing this enables the remote Hazelcast Session module and any dependent modules or files needed for it to run on the server.
+The example above is using a fresh `${jetty.base}` with nothing else enabled.
+Because Hazelcast is not a technology provided by the Eclipse Foundation, users are prompted to assent to the licenses of the external vendor (Apache in this case).
+
+When the `--add-to-start` argument was added to the command line, it enabled the the `session-store-hazelcast-remote` module as well as the `sessions` and `server` modules, which are required for Hazelcast session management to operate.
+It also downloaded the needed Hazelcast-specific jar files and created a directory named `${jetty.base}/lib/hazelcast/` to house them.
+
+In addition to adding these modules to the classpath of the server it also added several ini configuration files to the `${jetty.base}/start.d` directory.
+
+____
+[NOTE]
+If you have updated versions of the jar files automatically downloaded by Jetty, you can place them in the associated `${jetty.base}/lib/` directory and use the `--skip-file-validation=<module name>` command line option to prevent errors when starting your server.
+____
+
+==== Configuring Hazelcast Remote Properties
+
+Opening the `start.d/session-store-hazelcast-remote.ini` will show a list of all the configurable options for the Hazelcast module:
+
+[source, screen, subs="{sub-order}"]
+----
+# ---------------------------------------
+# Module: session-store-hazelcast-remote
+# Enables session data store in a remote Hazelcast Map
+# ---------------------------------------
+--module=session-store-hazelcast-remote
+
+#jetty.session.hazelcast.jettySessionMapName=jetty_sessions
+#jetty.session.hazelcast.onlyClient=true
+#jetty.session.hazelcast.configurationLocation=
+#jetty.session.gracePeriod.seconds=3600
+#jetty.session.savePeriod.seconds=0
+----
+
+jetty.session.hazelcast.jettySessionMapName::
```

Comment:
Can you please make sure to add this .adoc to the[ chapter.adoc](https://github.com/eclipse/jetty.project/blob/jetty-9.4.x/jetty-documentation/src/main/asciidoc/administration/sessions/chapter.adoc) so it is included when the documentation is built?